terminology_bot/Dockerfile

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
It's look like you create your own container with pg based on the ubuntu. Is it necessary?

We might use something like https://hub.docker.com/_/postgres/

Reply:
As for storing data in DB, there was such a requirement from Rasul.

Reply:
As for postgres official image, yes, we may use it.

Reply:
Definitely should use Postgres image.